tfidf-benchmark/NLP-pipeline-spark.ipynb

```diff
@@ -0,0 +1,673 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #16.                        data = spark.read.parquet(parquet_path).sample(withReplacement=False, fraction=data_frac, seed=seed)
</pre><p>We need to persist this too to benchmark read_parquet correctly. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/rapids-examples/pull/31/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>Yup makes sense. </p>